source/dataaccess.rst

```diff
@@ -0,0 +1,27 @@
+============================================
+Data Access for advanced use
+============================================
+
+-------------------------------------------------------------
+Functionality
+-------------------------------------------------------------
+* Users should be able to produce a dataset and upload to the data access service without time-consuming transformations.
+* Users must be able to download a copy of the entire dataset.
+* Users must be able to stream and filter parts of the dataset.
+* Users need access to enough metadata to be able to locally post-process, reproject etc. the dataset.
+* Data access service need to support the requirements for DOI: <ref>.
+* Data access service need to be FAIR.
```

Comment:
Forslag: "The data access service must support the FAIR principles, in particular (meta)data interoperability and reusability"